File: leetcode/problems/typescript/1768. Merge Strings Alternately/solve.ts
function mergeAlternately(word1: string, word2: string): string {
  const MaxLen = Math.max(word1.length, word2.length);
  let merged = "";
  for (let i = 0; i < MaxLen; i++) {
    merged += (word1[i] ?? "") + (word2[i] ?? "");
  }
  return merged;
}

File: leetcode/problems/python3/1768. Merge Strings Alternately/solve.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        DL = min(len(word1), len(word2))
        return (
            "".join(map(lambda x: x[0] + x[1], zip(word1, word2)))
            + word1[DL:]
            + word2[DL:]
        )
